src/ui/gui.css

```diff
@@ -118,7 +123,6 @@ table.spaced {
 
 table.dice {
     padding: 0px;
-    spacing: 0px;
```

Comment:
Not sure what to do about this --- it adds a bunch of visual testing of post-login behavior to this pull.  What's easier for you, if i add that testing time onto this pull, or if we do it separately?  I'm happy either way.